src/components/tooltip/daikin-tooltip.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  autoUpdate,
+  computePosition,
+  flip,
+  offset,
+  shift,
+} from "@floating-ui/dom";
+import { cva } from "class-variance-authority";
+import { css, html, LitElement, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { createRef, ref, type Ref } from "lit/directives/ref.js";
+import tailwindStyles from "../../tailwind.css?inline";
+
+const cvaTooltip = cva(
+  [
+    "absolute",
+
+    "p-3",
+    "justify-center",
+    "items-center",
+    "rounded",
+    "w-max",
+    "top-0",
+    "left-0",
+    "max-w-[312px]",
+
+    "text-sm",
+    "font-daikinSerif",
+    "font-normal",
+    "not-italic",
+    "leading-5",
+  ],
+  {
+    variants: {
+      variant: {
+        light: [
+          "border",
+          "border-solid",
+          "border-daikinNeutral-800",
+          "bg-[#ffffffe6]",
+          "text-black",
+        ],
+        dark: ["bg-[#414141e6]", "text-white"],
+      },
+      open: {
+        true: ["visible", "opacity-100"],
+        false: ["invisible", "opacity-0"],
+      },
+    },
+  }
+);
+
+/**
+ * A tooltip component.
+ *
+ * @fires click - Emitted when the trigger element be clicked.
+ * @fires keydown - Emitted when the key press down on trigger element.
+ * @fires mouseleave - Emitted when the mouse cursor move leave from trigger element
+ * @fires mouseenter - Emitted when the mouse cursor move enter trigger element
```

Comment:
`@fires` describes the events that cause the component to fire.
In this case, the tooltip component does not fire any events, so `@fires` is not necessary.
(Although I suppose it would be fine to fire events such as "open" and "close".)

```suggestion
 * @slot - A slot for the element to which the tooltip is attached (the trigger element).
 * @slot description - A slot for the tooltip content.
```

Reply:
BTW I thought `tooltip` is a better slot name than `description`.